atoma-state/src/migrations/20250411093352_add-tx-digest-to-stacks.sql

```diff
@@ -0,0 +1,8 @@
+-- Add tx_digest column to stacks table
+ALTER TABLE
+	stacks
+ADD
+	COLUMN tx_digest TEXT;
```

Comment:
Do we want to add a default value to be empty string ? and make it a required value

Reply:
Good point done in 24c7dcf